person.rb

```diff
@@ -19,6 +20,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
**[OPTIONAL BUT REQUIRED]** Hey, great making the changes as requested by the reviewer let's also add the books and rentals in the getter/setters method as the person would rent many books so they can be accessible. :point_right: `attr_accessor :name, :age, :books, :rentals`



**[REQUIRED]**  :point_right:  After adding the books, we also need to add the method for the rentals for every book rented by the person since the rentals in an intermediary between a person and a book to establish the has-many relationship. as you've rightly added, let's improve on the method further below:

```ruby
def add_rental(book, date)
    Rental.new(date, self, book)
  end
```
- The date is included to keep track of the date each book is rented and also, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association` which is why the self is added in the parameter.